sdks/python/container/license_scripts/dep_urls_py.yaml

```diff
@@ -77,8 +77,6 @@ pip_dependencies:
     license: "https://raw.githubusercontent.com/mtth/hdfs/master/LICENSE"
   httplib2:
     license: "https://raw.githubusercontent.com/httplib2/httplib2/master/LICENSE"
-  mmh3:
-    license: "https://raw.githubusercontent.com/hajimes/mmh3/master/LICENSE"
```

Comment:
so pip-licences tool is able to automatically fetch the license for mmh3? If so, great! If not, we will need this line, since we will ship mmh3 in our containers.

Reply:
hmm probably not-- will add it back in